### PR TITLE
Tickets/cap 511

### PR DESCRIPTION
**Pull request by tony-johnson**
This pull requests satisfies CAP-511 and cap-515 (see comments in JIRAs).
I somewhat accidentally reformatted sal_interfaces/CCCamera/CCCamera_Events.xml (all of our XML files should be reformatted to follow proper indentation) so github shows the file as one big change. If you feel so inclined you can use 

git diff --ignore-all-space develop...tickets/CAP-511

(I don't know if it possible to get github to do that).

### CODE REVIEW

**Comment by tony-johnson** [comment]
Can you give an example of something which should not be unitless?

File: sal_interfaces/CCCamera/CCCamera_Events.xml [hunk 1]
```diff
@@ -1,1128 +1,1407 @@
...
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_awGain</EFDB_Name>
+            <Description>anti-windup gain</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_basePower</EFDB_Name>
```
Comment (mareuter) [inline sal_interfaces/CCCamera/CCCamera_Events.xml:1192]: Is this in watts?

File: sal_interfaces/CCCamera/CCCamera_Events.xml [hunk 2]
```diff
@@ -1,1128 +1,1407 @@
...
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_gain</EFDB_Name>
+            <Description>loop gain</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_maxInput</EFDB_Name>
```
Comment (mareuter) [inline sal_interfaces/CCCamera/CCCamera_Events.xml:1206]: Is this a temperature, voltage or current input? Needs units.

File: sal_interfaces/CCCamera/CCCamera_Events.xml [hunk 3]
```diff
@@ -1,1128 +1,1407 @@
...
+            <Description>maximum input</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_maxOutput</EFDB_Name>
```
Comment (mareuter) [inline sal_interfaces/CCCamera/CCCamera_Events.xml:1213]: Shows watts in the comments but labelled as unitless. Typo?

File: sal_interfaces/CCCamera/CCCamera_Events.xml [hunk 4]
```diff
@@ -1,1128 +1,1407 @@
...
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_minInput</EFDB_Name>
+            <Description>minimum input</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_minOutput</EFDB_Name>
```
Comment (mareuter) [inline sal_interfaces/CCCamera/CCCamera_Events.xml:1227]: Ditto from above.

File: sal_interfaces/CCCamera/CCCamera_Events.xml [hunk 5]
```diff
@@ -1,1128 +1,1407 @@
...
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_rebs</EFDB_Name>
+            <Description>The REBs with heaters to use</Description>
+            <IDL_Type>string</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_setTemp</EFDB_Name>
```
Comment (mareuter) [inline sal_interfaces/CCCamera/CCCamera_Events.xml:1241]: Definitely a temperature but needs proper units.

File: sal_interfaces/CCCamera/CCCamera_Events.xml [hunk 6]
```diff
@@ -1,1128 +1,1407 @@
...
+            <Description>Temperature set point</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_smoothTime</EFDB_Name>
```
Comment (mareuter) [inline sal_interfaces/CCCamera/CCCamera_Events.xml:1248]: Marked as a time but is unitless. Should have time units.

File: sal_interfaces/CCCamera/CCCamera_Events.xml [hunk 7]
```diff
@@ -1,1128 +1,1407 @@
...
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_tempChans</EFDB_Name>
+            <Description>Temperature channels to use</Description>
+            <IDL_Type>string</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>raft_TempControl_timeConst</EFDB_Name>
```
Comment (mareuter) [inline sal_interfaces/CCCamera/CCCamera_Events.xml:1262]: Says (secs) in comment so should that be seconds for units?

Comment (tony-johnson) [reply sal_interfaces/CCCamera/CCCamera_Events.xml:1262]: OK, thanks, these are now generated automatically from CCS, so the real problem is that the units are missing on the original CCS source. I fixed some already but I missed these ones. Will fix it in the XML.

Comment (tony-johnson) [reply sal_interfaces/CCCamera/CCCamera_Events.xml:1262]: Units Fixed.

**Pull request merged**